pages/common/ftp.md

```diff
@@ -0,0 +1,27 @@
+# ftp
+
+> Internet File Transfer Protocol
+
+- Connect to a FTP server:
```

Comment:
@waldyrious is our in-house grammar expert, if you haven't noticed. :wink: 

btw - Can someone explain the reason for the "an", for plebs like me ?